src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 900;
+  font-style: normal;
+  color: #3d4e61;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+}
+
+.search {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: -1px;
+}
+
+.search-box {
+  position: relative;
+
+}
+
+.search-box.small {
+  position: relative;
+}
+
+.input-small {
+  height: 42px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 33px;
+  line-height: 19px;
+}
+
+.search-small {
+  position: absolute;
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 3px;
+}
+
+.input-small:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input-small:focus {
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
```

Comment:
You repeat this value for many classes. Move it to the html selector and only inherit it in those classes in which it is necessary. Fix it everywhere